lux/action/similarity.py

```diff
@@ -17,9 +17,10 @@
 import math
 import numpy as np
 from lux.vis.VisList import VisList
+from lux.utils.utils import get_filter_specs
 
 
-def similar_pattern(ldf, intent, topK=-1):
+def similar_pattern(ldf, *args):
```

Comment:
Merge with `Filter` action and move the similarity metric into an interestingness metric